<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1523</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>1011</width>
     <height>534</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="dataVizWidget" native="true"/>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="3">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="PyDMLineEdit" name="StepsInput1">
          <property name="toolTip">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QComboBox" name="DetectorSelector1"/>
        </item>
        <item row="0" column="0">
         <widget class="QComboBox" name="MotorSelector1"/>
        </item>
        <item row="0" column="1">
         <widget class="PyDMLineEdit" name="StartInput1">
          <property name="toolTip">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="PyDMLineEdit" name="StopInput1">
          <property name="toolTip">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMLineEdit</class>
   <extends>QLineEdit</extends>
   <header>pydm.widgets.line_edit</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
